code/modules/outfits/outfit.js

```diff
@@ -23,9 +23,11 @@ class Outfit {
 				target.c.MobInventory.slots.iclothing.equip_or_del(new Atom(target.server, this.slots.iclothing));
 			if(this.slots.oclothing && target.c.MobInventory.slots.oclothing)
 				target.c.MobInventory.slots.oclothing.equip_or_del(new Atom(target.server, this.slots.oclothing));
+			if(this.slots.head && target.c.MobInventory.slots.head)
+				target.c.MobInventory.slots.head.equip_or_del(new Atom(target.server, this.slots.head));
 
 			for(let [id, slot] of Object.entries(target.c.MobInventory.slots)) {
-				if(id == "iclothing" || id == "oclothing")
+				if(id == "iclothing" || id == "oclothing" || id == "head")
```

Comment:
This is unnecessary.

The purpose of this bit of code is to ensure that uniform is the *first* thing equipped, and the outer clothing is *second*. This is because those two tend to have slots that depend on their existence, such as ID slots, pocket slots, and for the latter the suit storage slot. However, there is no such slots for the head, making this unnecessary.